### PR TITLE
redundans: add 0.14a, cleanup install

### PR DESCRIPTION
**Pull request by molecuul**
couldn't find an actual release tarball for 0.14a but several commits which seem to indicate it,
cleaned up the install process if new files are added in the future.

older release marked as preferred over the alpha release.

### CODE REVIEW

File: var/spack/repos/builtin/packages/redundans/package.py [hunk 1]
```diff
@@ -11,8 +11,11 @@ class Redundans(Package):
 
     homepage = "https://github.com/Gabaldonlab/redundans"
     url      = "https://github.com/Gabaldonlab/redundans/archive/v0.13c.tar.gz"
+    git      = "https://github.com/Gabaldonlab/redundans.git"
 
-    version('0.13c', '2003fb7c70521f5e430553686fd1a594')
+    version('0.14a', commit='a20215a862aed161cbfc79df9133206156a1e9f0')
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/redundans/package.py]: I can't tell if this is actually an alpha release or if they just use a, b, c for the minor version number. They seem to have had an `alpha` release in the past. I think it's fine to put this as the preferred version.

File: var/spack/repos/builtin/packages/redundans/package.py [hunk 2]
```diff
@@ -33,12 +36,16 @@ def install(self, spec, prefix):
         filter_file(r'sspacebin = os.path.join(.*)$',
                     'sspacebin = \'' + sspace_location + '\'',
                     'redundans.py')
-        install('redundans.py', prefix.bin)
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/redundans/package.py]: I don't think this file is present in the `bin` directory.

File: var/spack/repos/builtin/packages/redundans/package.py [hunk 3]
```diff
@@ -33,12 +36,16 @@ def install(self, spec, prefix):
...
+
+        binfiles = ['fasta2homozygous.py', 'fasta2split.py',
+                    'fastq2insert_size.py', 'fastq2mates.py',
+                    'fastq2shuffled.py', 'fastq2sspace.py',
+                    'filterReads.py', 'redundans.py']
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/redundans/package.py]: What other files are in this directory? Could you use `install_tree`? If there are non-Python files, could you use glob to locate the files to install?

File: var/spack/repos/builtin/packages/redundans/package.py [hunk 4]
```diff
@@ -33,12 +36,16 @@ def install(self, spec, prefix):
...
+
+        # new internal dep with 0.14a
+        if spec.satisfies('@0.14a:'):
+            binfiles.extend(['denovo.py'])
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/redundans/package.py]: Use `append()`, not `extend([])`.

File: var/spack/repos/builtin/packages/redundans/package.py (hunk 3 above)
Comment (molecuul) [reply var/spack/repos/builtin/packages/redundans/package.py]: There are some skeleton files for dependencies which are provided by depends_on -- including them in the install overrides spack's packages

File: var/spack/repos/builtin/packages/redundans/package.py (hunk 1 above)
Comment (baberlevi) [reply var/spack/repos/builtin/packages/redundans/package.py]: Without a tagged release, I think we should just add a development version pointing to the tip of the master branch.  Besides the change log & a few places in code, this is the only reference I can find to that version: https://groups.google.com/forum/#!searchin/redundans/14a%7Csort:date/redundans/XYeEt9Mys1w/Xv_goBRPAgAJ

**Comment by scheibelp** [comment]
Given the announcement link in https://github.com/spack/spack/pull/10006/files#r238691840, I think treating the git commit as a release is suitable.

**Pull request merged**